graphql/schema.graphql

```diff
@@ -1292,11 +1292,11 @@ type WorkstationConfig {
 type BuildBaronSettings {
   ticketCreateProject: String!
   ticketSearchProjects: [String!]
-  bfSuggestionServer: String!
-  bfSuggestionUsername: String!
-  bfSuggestionPassword: String!
-  bfSuggestionTimeoutSecs: Int!
-  bfSuggestionFeaturesURL: String!
+  bfSuggestionServer: String
+  bfSuggestionUsername: String
+  bfSuggestionPassword: String
+  bfSuggestionTimeoutSecs: Int
+  bfSuggestionFeaturesURL: String
 }
 
```

Comment:
Sorry forgot to mention -- `PLUGINS` must also be added to the ProjectSettingsSection enum in this file. Thanks!